swarm.cabal

```diff
@@ -127,28 +128,34 @@ library
                       aeson                         >= 2 && < 2.1,
                       array                         >= 0.5.4 && < 0.6,
                       brick                         >= 1.0 && < 1.1,
+                      bytestring                    >= 0.10 && < 0.11,
                       clock                         >= 0.8.2 && < 0.9,
                       containers                    >= 0.6.2 && < 0.7,
                       directory                     >= 1.3 && < 1.4,
                       dotgen                        >= 0.4 && < 0.5,
                       either                        >= 5.0 && < 5.1,
+                      extra                         >= 1.7 && < 1.8,
```

Comment:
Some functions like `breakOnEnd` that `Text` has, but `[]` does not.

It would be easy to avoid, but I dislike converting to `Text` and back again just to get a few simple functions.

The `extra` package has a negligible dependency footprint and might already be transitively included. :shrug: